ts/features/wallet/component/features/PaymentMethodInitiatives.tsx

```diff
@@ -31,18 +32,23 @@ const PaymentMethodInitiatives = (props: Props): React.ReactElement | null => {
 
   const dispatch = useIODispatch();
 
-  const refresh = React.useCallback(() => {
-    dispatch(
-      idPayInitiativesFromInstrumentRefreshStart({
-        idWallet: idWalletString
-      })
-    );
-    return () => {
-      dispatch(idPayInitiativesFromInstrumentRefreshStop());
-    };
-  }, [idWalletString, dispatch]);
-
-  useFocusEffect(refresh);
+  useFocusEffect(
+    React.useCallback(() => {
+      dispatch(
+        idPayInitiativesFromInstrumentGet.request({
+          idWallet: idWalletString
+        })
+      );
+      dispatch(
+        idPayInitiativesFromInstrumentRefreshStart({
+          idWallet: idWalletString
+        })
+      );
+      return () => {
+        dispatch(idPayInitiativesFromInstrumentRefreshStop());
+      };
+    }, [idWalletString, dispatch])
```

Comment:
What do you think to wrap-up the entire callback into a function and invoking it inside the userFocusEffect?

Reply:
See https://github.com/pagopa/io-app/pull/5279/commits/287afdba8254dcf60f1146546ee08df0361887f1